recipe/build.sh

```diff
@@ -1,4 +1,16 @@
 #!/bin/bash
 
-cmake ${CMAKE_ARGS} -DEXPECTED_ENABLE_TESTS=OFF -DCMAKE_INSTALL_PREFIX=$PREFIX $SRC_DIR -DCMAKE_INSTALL_LIBDIR=lib
-make install
+# Isolate the build.
+mkdir -p Build
+cd Build || exit 1
+
+# Generate the build files.
+cmake .. -G"Ninja" ${CMAKE_ARGS} \
+    -DEXPECTED_ENABLE_TESTS=OFF \
```

Comment:
Same comment as `bld.bat`: try to run the author's tests.

Reply:
Added the tests (Catch2). I had to replace the catch2 header for osx arm64 to a newer version. Otherwise it would assume osx was x86 and fail to compile.